himportscan/src/HImportScan/ImportScanner.hs

```diff
@@ -98,7 +99,7 @@ scanImports dynFlags filePath contents = do
     -- This is far from ideal, however handling this better would require being able to communicate errors better to go.
     Left err -> do
       GHC.printBagOfErrors dynFlagsWithExtensions err
-      error "ghc parsing failed"
+      throwIO (GHC.mkSrcErr err)
```

Comment:
Does this commit fix the issue where users don't know what parsing failed?

Reply:
No, it doesn't. I tested this change before writing about it.

Reply:
Right now, after my change which splits stderr and stdout, we get the parse error that ghc reports, e.g.
```


/home/googleson78/git/gazelle_haskell_modules/example/package-a/app/Main.hs:9:59: error:
    parse error on input `PatTrue'
  |
9 | import PackageA.Other.C(type DataC, DataC(DataC), pattern PatTrue)
  |                                                           ^^^^^^^
himportscan: parse error on input `PatTrue'


gazelle: himportscan exited unsuccessfully exit status 1
```
Is this good enough, or should we explicitly report the file name separately?